src/libsync/propagatorjobs.cpp

```diff
@@ -189,6 +194,7 @@ void PropagateLocalMkdir::startLocalMkdir()
     // before the correct etag is stored.
     SyncFileItem newItem(*_item);
     newItem._etag = "_invalid_";
+    qCInfo(lcPropagateLocalMkdir) << "Created folder:" << newDirStr << " updating the metadata for newItem._file:" << newItem._file << "newItem._type:" << newItem._type << " QFileInfo(newDirStr).isDir():" << QFileInfo(newDirStr).isDir();
```

Comment:
debug please

Reply:
@mgallien Not sure why debug is better? In case this is for sake of not having extra stuff in the release configuration, we still have the debug log level enabled by default. But, if the debug log level is disabled for that user somehow, I then may lose these lines and this could prevent me from seeing the entire picture.